arrows/core/transform_detected_object_set.cxx

```diff
@@ -239,18 +240,11 @@ vital::bounding_box<double>
 transform_detected_object_set::
 transform_bounding_box( vital::bounding_box<double>& bbox ) const
 {
-  vector_4d bbox_bounds;
-  bbox_bounds << bbox.min_x(), bbox.min_y(),
-                 bbox.max_x(), bbox.max_y();
-
-  vector_4d out_bounds = this->view_to_view( this->src_camera,
-                                             this->dest_camera,
-                                             bbox_bounds );
-
-  return vital::bounding_box<double>(out_bounds(0),
-                                     out_bounds(1),
-                                     out_bounds(2),
-                                     out_bounds(3));
+  vital::bounding_box<double> out_bbox = this->view_to_view( this->src_camera,
```

Comment:
Here is another case where you can eliminate the temporary variable a directly return the result of the function call.